bootstrap/namespaces/helmfile/values/namespaces-wc.yaml.gotmpl

```diff
@@ -9,6 +9,7 @@ namespaces:
 - name: monitoring
 - name: ingress-nginx
 - name: velero
+- name: gatekeeper-system
```

Comment:
Does this just work even though this is also listed inside the `if opa enabled`?